client/chatbotWidget/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
```

Comment:
Here is where we'll need to add the following line for ant-d: 
import 'antd/dist/reset.css'

Reply:
Nice, you can do that when you create the widget.